src/core/rateLimiter.js

```
@@ -252,19 +267,79 @@ class RateLimiter {
+        // Clear tokens
         this.tokens = { github: null, gitlab: null };
     }
+
+    // Format time in MM:SS format
+    formatTime(seconds) {
+        const mins = Math.floor(seconds / 60);
+        const secs = seconds % 60;
+        return `${mins.toString().padStart(2, '0')}:${secs.toString().padStart(2, '0')}`;
+    }
+
+    // Start countdown timer for rate limit reset
```

Biome (1.9.4)

[error] Duplicate class member name "formatTime"



(lint/suspicious/noDuplicateClassMembers)